vim/core/oni-core-interop/plugin/init.vim

```diff
@@ -110,10 +110,7 @@ let context.windowTopLine = line("w0")
 let context.windowBottomLine = line("w$")
 let context.byte = line2byte(line(".")) + col(".")
 let context.filetype = eval("&filetype")
-
-if exists("b:last_change_tick")
```

Comment:
yeah, I didn't realize the thing about the text document item, it seems weird to me that they check the version and not if it's modified, but either way, I will fix it.